### DISCUSSION TITLE
Bug? Traffic graph shows own activity

### DISCUSSION THREAD

**Discussion by attah** [community/community#11619] [Other Feature Feedback, Questions, & Ideas]
I recently noticed that the traffic graph seems to have started including my own (logged in) visits to my own repos.
It didn't use to do that, and it dilutes the data unnecessarily - so surely it cannot be intentional? Is it?

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by attah** [reply]
Thanks; would have been great if that was more "findable", i was looking like crazy.

**Reply by MathiasReker** [reply]
You are welcome :)
